### PR TITLE
4878 simple r install

### PR DESCRIPTION
**Pull request by landreev**
## New Contributors

Welcome! New contributors should at least glance at [CONTRIBUTING.md](/CONTRIBUTING.md), especially the section on pull requests where we encourage you to reach out to other developers before you start coding. Also, please note that we measure code coverage and prefer you write unit tests. Pull requests can still be reviewed without tests or completion of the checklist outlined below. Thanks!

## Related Issues

- connects to #4878: Provide "bare minimum" R installation instructions; add "haven" package.

## Pull Request Checklist

- [ ] Unit [tests][] completed
- [ ] Integration [tests][]: None
- [ ] Deployment requirements, [SQL updates][], [Solr updates][], etc.: None
- [ ] [Documentation][docs] completed
- [ ] Merged latest from "develop" [branch][] and resolved conflicts

[tests]: http://guides.dataverse.org/en/latest/developers/testing.html
[SQL updates]: https://github.com/IQSS/dataverse/tree/develop/scripts/database/upgrades
[Solr updates]: https://github.com/IQSS/dataverse/blob/develop/conf/solr/7.3.0/schema.xml
[docs]: http://guides.dataverse.org/en/latest/developers/documentation.html
[branch]: http://guides.dataverse.org/en/latest/developers/branching-strategy.html

### CODE REVIEW

**Comment by donsizemore** [comment]
looks good from Odum's end!
[https://nevermore.irss.unc.edu/ddi-explorer/?fileId=7503696&siteUrl=https://dataverse-test.irss.unc.edu#?selected=@_id%3Dv39973600%26type%3Drow;%26_id%3Dv39973606%26type%3Drow;%26_id%3Dv39973625%26type%3Drow&view=chart](https://nevermore.irss.unc.edu/ddi-explorer/?fileId=7503696&siteUrl=https://dataverse-test.irss.unc.edu#?selected=@_id%3Dv39973600%26type%3Drow;%26_id%3Dv39973606%26type%3Drow;%26_id%3Dv39973625%26type%3Drow&view=chart)

File: src/main/java/edu/harvard/iq/dataverse/ingest/tabulardata/impl/plugins/rdata/RDATAFileReader.java [hunk 1]
```diff
@@ -90,7 +90,7 @@
   // RServe static variables
   private static String RSERVE_HOST = System.getProperty("dataverse.rserve.host");
   private static String RSERVE_USER = System.getProperty("dataverse.rserve.user");
-  private static String RSERVE_PASSWORD = System.getProperty("dataverse.rserve.pwrd");
+  private static String RSERVE_PASSWORD = System.getProperty("dataverse.rserve.password");
```
Comment (pdurbin) [inline src/main/java/edu/harvard/iq/dataverse/ingest/tabulardata/impl/plugins/rdata/RDATAFileReader.java]: Nice to see this `dataverse.rserve.password` fix rolled in.

**Comment by landreev** [comment]
@donsizemore Please note that TwoRavens also appears to be working properly on irss-rservetest now. It's currently configured to work with my own test Dataverse here at Harvard.

File: doc/sphinx-guides/source/installation/r-rapache-tworavens.rst [hunk 2]
```diff
@@ -238,7 +254,7 @@ b. Rename the resulting directory "dataexplore" ...
 
 ...and place it in the web root directory of your apache server. We'll assume ``/var/www/html/dataexplore`` in the examples below::
 
-        mv TwoRavens-a6869eb28693d6df529e7cb3888c40de5f302b66 /var/www/html/dataexplore
+        mv TwoRavens-dataverse-distribution /var/www/html/dataexplore
```
Comment (pdurbin) [inline doc/sphinx-guides/source/installation/r-rapache-tworavens.rst]: I guess this is better than a6869eb? I guess we want to be able to change or fix the distributions of TwoRavens. Longer term, the TwoRavens team should tag their releases with version numbers.

**Comment by donsizemore** [comment]
Excellent! I just ran through the prerequisite installations, and Data Explorer works properly. A couple of notes which may or may not make things easier for end-users:

- CentOS 7 includes the epel-release in its base repository; only RHEL7 users will need to copy-paste the URL. May be simpler to leave the instructions as-is for both, though.
- a shorter package installation line might be:

```
install.packages(c("R2HTML","rjson","DescTools","Rserve","haven"))
```

which on my CentOS 7 system defaults to /usr/lib64/R/library since I didn't specify a library location, or we could specify it and the repo URL as additional arguments after the packagename vector. Just proposing a one-liner for end-users.

Thank you again @landreev , great to have R 3.5 in gear!

File: scripts/r/rserve/rserve-setup.sh [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    echo "You can start Rserve daemon by executing"
+    echo "  service rserve start"
+    echo 
+    echo "If this is a RedHat/CentOS 7 system, you may want to use the systemctl file rserve.service instead (provided in this directory)"
+else 
+    echo "Rserve startup file already in place."
+fi
+
+echo 
+echo "Successfully installed Dataverse Rserve framework."
```
[the remark this answers is not in the document]
Comment (landreev) [reply scripts/r/rserve/rserve-setup.sh]: Would be very simple to add this to vagrant, yes. What this script does is super straightforward/non-controversial.

**Pull request merged**